activerecord/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 ## Rails 7.1.0.beta1 (September 13, 2023) ##
 
+*   Dump PostgreSQL schemas as part of the schema dump.
+
+    *Lachlan Sylvester*
+
```

Comment:
This commit is not included in Rails 7.1.0.beta1, so would you move this entry to the top of `activerecord/CHANGELOG.md`?

Reply:
@yahonda I have done this. Thanks for pointing this out.